### PR TITLE
tccutil 1.2.0 (new formula)

### PR DESCRIPTION
**Pull request by reitermarkus**

### CODE REVIEW

File: Library/Formula/tccutil.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  head "https://github.com/jacobsalmela/tccutil.git"
+
+  bottle :unneeded
+
+  def install
+    bin.install "tccutil.py" => "tccutil"
+  end
+
+  test do
+    system "#{bin}/tccutil", "--help"
```
Comment (mikemcquaid) [inline Library/Formula/tccutil.rb]: Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

File: Library/Formula/tccutil.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  desc "Utility to modify the OS X Accessibility Database (TCC.db)"
+  homepage "https://github.com/jacobsalmela/tccutil"
+  url "https://github.com/jacobsalmela/tccutil/archive/v1.2.0.tar.gz"
+  sha256 "3fcac0c458969629366002a2e0142a7291ab44a0077c5495de4b1383704861bc"
+  head "https://github.com/jacobsalmela/tccutil.git"
+
+  bottle :unneeded
+
+  def install
+    bin.install "tccutil.py" => "tccutil"
```
Comment (mikemcquaid) [inline Library/Formula/tccutil.rb]: What version of Python does this need?

Comment (reitermarkus) [inline Library/Formula/tccutil.rb]: It is designed to replace the built-in OS X utility, so it runs fine without dependencies.

File: Library/Formula/tccutil.rb (hunk 1 above)
Comment (reitermarkus) [inline Library/Formula/tccutil.rb]: No, sorry. Even simply listing entries in the database requires `sudo`.

File: Library/Formula/tccutil.rb (hunk 2 above)
Comment (mikemcquaid) [inline Library/Formula/tccutil.rb]: In that case this should be either `:keg_only` or live in `homebrew/dupes` as it's shadowing a system utility.

Comment (reitermarkus) [inline Library/Formula/tccutil.rb]: Ok, then I'll resubmit it to `homebrew/dupes`.

Comment (mikemcquaid) [inline Library/Formula/tccutil.rb]: I'm :ok: with it being `keg_only` if you are as technically (correct me if I'm wrong) this isn't the same version as the one shipped with the system?

Comment (reitermarkus) [inline Library/Formula/tccutil.rb]: I should add that technically it is not replacing but rather extending the built-in utility. The built-in utility only supports one command: `reset`.

This command is passed through by the utility.

If this doesn't change anything, I'd still rather add it to `homebrew/dupes` as I think it's clearer that way.

Comment (mikemcquaid) [inline Library/Formula/tccutil.rb]: So `reset` is passed through to the other utility as-is? Given this requires `sudo` too which we don't use during our build process I think I'll just accept it as-is. Thanks!

Comment (reitermarkus) [inline Library/Formula/tccutil.rb]: Exactly, `reset` is passed through to OS X's built-in utility, doesn't require `sudo` though. (To be clear: The built-in utility also doesn't need `sudo`.)

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**